### DISCUSSION TITLE
Known Alternativ

### DISCUSSION THREAD

**Discussion by CarliJoy** [josephernest/nfreezer#11] [General]
I friend just recommended me: https://borgbackup.readthedocs.io/en/stable/
Maybe you can put that in you comparison Table as well :-)

But it seems to have the same feature set you requested.

**Comment by josephernest** [comment]
Hi @CarliJoy, 

Thanks for your feedback!
Yes when I have time, I'll try Duplicati (which is not the same than Duplicity) (see https://github.com/josephernest/nfreezer/issues/6), Restic (https://github.com/josephernest/nfreezer/issues/5) and the one you mentioned. They all seem to have a similar feature set indeed.

The main goal of nFreezer is to be able to do this and stay very simple: single .py file, less than 300 lines of code; so that it's easy for anyone to read 100% of the source code before deciding to use it or not.

**Reply by CarliJoy** [reply]
Woha, there are a lot of tools available.
Thanks for the time!

**Reply by CarliJoy** [reply]
BTW: Here is already one Blog Post related: https://acha.ninja/blog/encrypted_backup_shootout/
